docs/src/main/sphinx/connector/faker.md

```diff
@@ -317,3 +338,44 @@ CREATE TABLE generator.default.customer (
   group_id INTEGER WITH (allowed_values = ARRAY['10', '32', '81'])
 );
 ```
+
+(faker-statistics)=
+### Using existing data statistics
+
+The Faker connector automatically sets the `default_limit` table property,
+and the `min`, `max`, `null_probability` column properties, based on statistics
+collected by scanning existing data, like in the following example:
+
+```sql
+CREATE TABLE generator.default.customer AS
+SELECT *
+FROM production.public.customer
+WHERE created_at > CURRENT_DATE - INTERVAL '1' YEAR;
+```
+
+Instead of using range, or other predicates, tables can be sampled,
+see [](tablesample).
+
+When the `SELECT` statement doesn't contain a `WHERE` clause, a shorter notation
+can be used:
+
+```sql
+CREATE TABLE generator.default.customer AS TABLE production.public.customer;
+```
+
+The Faker connector detects sequence columns, which are integer column with the
+number of distinct values almost equal to the number of rows in the table. For
+such columns, Faker sets the `step` column property to 1.
+
+Sequence detection can be turned off using the `sequence_detection_enabled`
+table, or schema property or in the connector configuration file, using the
+`faker.sequence-detection-enabled` property.
+
+The Faker connector detects dictionary columns, which are columns of
```

Comment:
Example?

Reply:
I added a complete example for this whole section, copying `tpch.tiny.orders`.

Reply:
BTW, I found a bug when creating this example: https://github.com/trinodb/trino/pull/24903